From f0c982f659842032d1beba320a51874d2f1186a8 Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Thu, 11 Aug 2016 16:42:21 -0700
Subject: [PATCH] joyent/node-verror#36 elide development-only files from npm
 package Reviewed by: Patrick Mooney <patrick.mooney@joyent.com> Approved by:
 Patrick Mooney <patrick.mooney@joyent.com>

---
 .npmignore | 9 +++++++++
 CHANGES.md | 1 +
 2 files changed, 10 insertions(+)
 create mode 100644 .npmignore

diff --git a/.npmignore b/.npmignore
new file mode 100644
index 0000000..f14aec8
--- /dev/null
+++ b/.npmignore
@@ -0,0 +1,9 @@
+.gitignore
+.gitmodules
+deps
+examples
+experiments
+jsl.node.conf
+Makefile
+Makefile.targ
+test
diff --git a/CHANGES.md b/CHANGES.md
index d481694..f6dc25c 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -2,3 +2,4 @@
 
 * #10 better support for extra properties on Errors
 * #11 make it easy to find causes of a particular kind
+* #36 elide development-only files from npm package
-- 
2.21.0

